php/php-7.4.32/wl-build.sh

```diff
@@ -30,6 +30,7 @@ export PHP_CONFIGURE='--without-libxml --disable-dom --without-iconv --without-o
 
 if [[ -v WASMLABS_RUNTIME ]]
 then
+    export LDFLAGS="-lwasmedge_sock ${LDFLAGS}"
```

Comment:
wasmedge_sock should be conditional, no?

Reply:
A very good catch!